backend/src/main/resources/db/migration/V2023.01.10.1146__create_schema.sql

```diff
@@ -0,0 +1 @@
+CREATE SCHEMA spar;
```

Comment:
I would highly suggest to always use `IF NOT EXISTS` to every SQL statement to prevent issues during migration.